src/item/held-item.ts

```diff
@@ -0,0 +1,28 @@
+import type { ElementalType } from "#enums/elemental-type";
+import { BaseItem } from "./base-item";
+
+/**
+ * Held items are items that are held onto by Pokemon
+ */
+export abstract class HeldItem extends BaseItem {
+  public isTransferable: boolean;
+  public ignorable: boolean;
```

Comment:
For consistency I’d prefix this with is too 
```suggestion
  public isIgnorable: boolean;
```

also check if it could be `readonly`